pkg/kubeinteraction/status/task_status.go

```diff
@@ -17,6 +18,8 @@ import (
 
 var reasonMessageReplacementRegexp = regexp.MustCompile(`\(image: .*`)
 
+const maxErrorSnippetCharacterLimit = 65535 // This is the maximum size allowed by Github check run logs and may apply to all other providers
+
```

Comment:
Apparently this length enforcement is in bytes, not unicode characters: https://github.com/github/docs/issues/35252 🫠